File: 2023_1_old/programmers/lv1/풀음/삼총사.py
def solution(number):
    cnt = 0
    number.sort()
    for i in range(len(number)):
        for j in range(i+1, len(number)):
            for k in range(j+1, len(number)):
                if number[i] + number[j] +number[k]== 0:
                    cnt += 1
                    print(number[i], number[j], number[k])
    return cnt

File: src/programmers/삼총사.java
class Solution {
    public int solution(int[] number) {
        int cnt =0;
        
        for(int i=0; i<number.length;i++){
            for(int j=i+1; j<number.length; j++){
                for(int w=j+1; w<number.length; w++){
                    if(number[i]+number[j]+number[w] ==0){
                        cnt ++;
                    }
                }
            }
        }
        
        return cnt;
        
    }
}
